config/v1/types_feature.go

```diff
@@ -217,9 +214,7 @@ var defaultFeatures = &FeatureGateEnabledDisabled{
 		vSphereControlPlaneMachineset,
 		sdnLiveMigration,
 	},
-	Disabled: []FeatureGateDescription{
-		disableKubeletCloudCredentialProviders, // We do not currently ship the correct config to use the external credentials provider.
```

Comment:
Needs to be moved into the Enabled list to be explicitly enabled

Reply:
So should it not be renamed `enableKubeletCloudCredentialProviders` to make more sense?

I'm also not following how we go from these changes to actually enabling / disabling functionality..